CRM/Xcm/Matcher/AddressMatcher.php

```diff
@@ -21,8 +21,15 @@ abstract class CRM_Xcm_Matcher_AddressMatcher extends CRM_Xcm_MatchingRule {
   protected $required_fields = array('street_address', 'postal_code', 'city');
   protected $additional_fields = NULL;
 
+  /**
+   * @var bool
+   */
+  protected $isStreetAddressParsingEnabled = FALSE;
```

Comment:
Even though it's not the case for the existing code, I think we should use PHP type hints instead of phpdoc type hints.

Reply:
This property is initialized in the constructor so it probably doesn't need a default value.

Reply:
I have changed this